index.html

```diff
@@ -9,138 +9,336 @@
-    </article>
+  <header>
+    <a href="#"><h2>John Doe</h2></a>
+    <nav>
+      <img src="menu.png" class="menu" alt="">
+      <ul class="menu_items">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+        <li><img src="mail.png" class="mail" alt="image"></li>
+      </ul>
+    </nav>
+  </header>  
 
-    <hr>
-
-    <section class="ellipse_19">
-      <figure class="img_placeholder">
-        <img src="placeholder.png" alt="">
-      </figure>
-
-      <article>
-        <h1 class="title_post">Multi-Post Stories</h1>
+  <main>
+    <section id="headline">
+      <div class="ellipse_18"></div>
+      <article class="headlines">
+        <h2> Owusu Dennis </h2>
+        <h3> Software Developer </h3>
       </article>
-  
-      <article>
+
+      <article class="brand">
         <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. has been the 
-          industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
+          I can help you build a product, 
+          feature or website Look through some of my 
+          work and experience! If you like what you 
+          see and have a project you need coded, don’t hestiate to contact me.
         </p>
       </article>
-  
-      <article class="tags">
-        <ul>
-          <li>css</li>
-          <li>html</li>
-          <li>javascript</li>
-          <li>Ruby</li>
-        </ul>
-      </article>
-  
-      <section class="button">See Projects</section>
-    </section>
-    <!--  -->
-    <section class="pro_2">
-      <figure class="img_placeholder">
-        <img src="placeholder.png" alt="">
-      </figure>
-      
-      <section class="ellipse_20"></section>
-      
-      <article>
-        <h1 class="title_post">Multi-Post Stories</h1>
-      </article>
-  
-      <article>
-        <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. has been the 
-          industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
+      <hr>
+    </section>  
+    
+    <section id="frame_container">
+      <article class="frame">
+        <section class="languages">
+          <h3>Languages</h3>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </section>
+
+        <div class="back"></div>
+        
+        <section class="frameworks">
+          <h3>Frameworks</h3>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </section>
       </article>
-  
-      <article class="tags">
+
+      <article class="skills">
+        <h3>Skills</h3>
```

Comment:
- [ ] I would suggest you, please change the color of `h3` tag which is: Language, Framework, and skills, and change the color of the verticle line as well. See the image below for reference:
### Your design
![his](https://user-images.githubusercontent.com/81727988/186726182-8d6620db-743b-40dd-bc46-a47c81ebb708.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/186726234-91522763-9636-4cf6-a5c2-332a1a03244c.png)